odl/solvers/util/callback.py

```diff
@@ -374,16 +375,47 @@ def __init__(self, *args, **kwargs):
         ----------
         display_step : positive int, optional
             Number of iterations between plots. Default: 1
+        saveto : string, optional
+            Path to a directory where the figures are to be saved, followed by
+            a file name and potentially a file format. Standard python string
+            formatting is possible to use in order to index the figures with
+            iteration number. If the directory name does not exist, a
+            ``ValueError`` is raised. If ``None``, the figures are not saved.
+            Default: ``None``
 
         Other Parameters
         ----------------
         kwargs :
             Optional arguments passed on to ``x.show``
+
+        Raises
```

Comment:
This does not actually raise, right? In that case we can actually drop this section.